README.md

```diff
@@ -321,10 +321,9 @@ const valid = (address.hashPart == hash);
 
 Parameters:
 
-* `source` (string, optional): sender address. for cases than source of transaction is important for dapp. wallet should check it or select from few accounts if wallet controls mora than one.
+* `source` (string, optional): sender address. For cases than source of transaction is important for dapp. wallet should check it or select from few accounts if wallet controls mora than one.
```

Comment:
```suggestion
* `source` (string, optional): sender address. Provided in case the source of transaction is important to the dapp. Wallet application must select the appropriate wallet contract to send the message from, or post an error if it does not have the keys to that specific address.
```